cdc/sorter/leveldb/system/system.go

```diff
@@ -211,37 +197,26 @@ func (s *System) Stop() error {
 	}
 	s.state = sysStateStopped
 
-	// TODO caller should pass context.
-	deadline := time.Now().Add(1 * time.Second)
-	ctx, cancel := context.WithDeadline(context.Background(), deadline)
-	defer cancel()
-	// Close actors
-	s.broadcast(ctx, s.DBRouter, message.StopMessage())
-	s.broadcast(ctx, s.WriterRouter, message.StopMessage())
-	s.broadcast(ctx, s.ReaderRouter, message.StopMessage())
-	s.broadcast(ctx, s.compactRouter, message.StopMessage())
-	// Close metrics goroutine.
-	close(s.closedCh)
-	// Wait actors and metrics goroutine.
-	s.closedWg.Wait()
-
-	// Stop systems.
-	err := s.dbSystem.Stop()
+	// Stop all actors and system to release resource.
+	err := s.WriterSystem.Stop()
```

Comment:
for `Stop` related logic, shall all `System` Stop() called, no matter the previous one success or not ?

Reply:
Good point, system.Stop never return error, I'll leave a TODO and fix it in a later PR.